packages/sdk-react/src/hooks/useTransactionReceipts.tsx

```diff
@@ -0,0 +1,35 @@
+import { useQuery } from '@tanstack/react-query';
+import { TransactionResponse } from 'fuels';
+
+import { useFuel } from '../components';
+import { QUERY_KEYS } from '../utils';
+
+export const useTransactionReceipts = ({ id }: { id?: string }) => {
```

Comment:
```suggestion
export const useTransactionReceipts = ({ txId }: { txId?: string }) => {
```
is this a txId or a messageId ?

Reply:
Do we even use this hook?

Reply:
I created because I was needing, but turn out I did in other way... but anyway the hook is useful so I kept it